mindmeld/bot/webex_bot_server.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module contains the Webex Bot Server component.
+"""
+
+import logging
+import json
+from flask import request
+import requests
+from ciscosparkapi import CiscoSparkAPI
+
+
+class WebexBotServer:
+    """
+    A sample server class for Webex Teams integration with any MindMeld application
+    """
+
+    def __init__(self, app, webhook_id, access_token, conv):
+        self.app = app
+        self.webhook_id = webhook_id
+        self.access_token = access_token
+        self.conv = conv
+
+        self.logger = logging.getLogger(__name__)
+
+        if not self.webhook_id or not self.access_token:
+            raise Exception('webhook_id and bot_access_token are not set')
+
+        self.spark_api = CiscoSparkAPI(self.access_token)
+        self.ACCESS_TOKEN_WITH_BEARER = 'Bearer ' + self.access_token
+        self.CISCO_API_URL = 'https://api.ciscospark.com/v1'
```

Comment:
These should be global constants that can be formatted based on the access_token